changelog.d/20240812_161617_mzhiltso_job_chunks.md

```diff
@@ -0,0 +1,4 @@
+### Added
+
+- A server setting to disable media chunks on the local filesystem
```

Comment:
```suggestion
- A server setting to disable permanent media chunks on local filesystem
```

Reply:
There were only permanent chunks on the local filesystem.

Reply:
Regular users (even those who may install self-hosted solution) do not know such implementation details.

Reply:
I just want to clarify that this is exactly about permanent chunks, explicitly

Reply:
Ok, but it's just what is passed in the task data endpoint API: `storage_method = 'file_system' | 'cache'`.

Reply:
Updated the message